pkgs/development/compilers/scala-runners/default.nix

```diff
@@ -0,0 +1,36 @@
+{ stdenv, fetchFromGitHub, jre, coursier }:
+
+let 
+  name = "scala-runners";
+  owner = "dwijnand";
+in stdenv.mkDerivation {
+  inherit name;
```

Comment:
```suggestion
stdenv.mkDerivation rec {
  name = "scala-runners";
```

Reply:
Also, instead of having just a `name`, packages should nowadays have a `pname` and a `version` (https://github.com/NixOS/nixpkgs/issues/103997)

Reply:
@raboof should I use (part of) the git tag? I can bother Dale if he wants to push tags but I think he may not care.

Reply:
there's no clear consensus on how to version packages where upstream doesn't use tags (#100833), using the (short) git tag seems ok to me in this case.

Reply:
@hrhino the convention I follow and I think is the best way to deal is

```
pname = "<name of software>";
version = "unstable-${date of the commit in the format YYYY-MM-DD}";
```  

On the other hand I would prefer `version = "<vcs used, say hg or git>-${date}"`